plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -82,6 +99,35 @@ func calculateResourceOccupancy(pod *api.Pod, podRequests *schedulercache.Resour
 	}
 }
 
+// Calculate the resource used on a node.  'node' has information about the resources on the node.
+// 'pods' is a list of pods currently scheduled on the node.
+// TODO: Use Node() from nodeInfo instead of passing it.
+func calculateUsedPriority(pod *api.Pod, podRequests *schedulercache.Resource, node *api.Node, nodeInfo *schedulercache.NodeInfo) schedulerapi.HostPriority {
```

Comment:
That's fine, but @mwielgus can you please add a comment to this function (calculateUsedPriority()), or to the place where you call this function from MostRequestedPriority() explaining why you do not just used LeastRequestedPriority/calculateUnusedPriority and reverse the sort order.